test/functional/test_ipmi_kcs.py

```diff
@@ -165,3 +166,40 @@ def test_qemu_local_user(self):
         print lines
         ssh.close()
         assert "admin" in lines
+
+    def test_smbios_data(self):
+        ssh = paramiko.SSHClient()
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        ssh.connect("127.0.0.1", port=2222, username="root", password="root",timeout=10)
+        stdin, stdout, stderr = ssh.exec_command("dmidecode -t1")
+        while not stdout.channel.exit_status_ready():
+            pass
+
+        lines = stdout.channel.recv(2048)
+        print lines
+
+        ssh.close()
+        with open("test.yml", "r") as yml_file:
+            self.conf= yaml.load(yml_file)
+
+        if self.conf["type"] == "quanta_d51":
+            assert "Manufacturer: Quanta Computer Inc" in lines
+            assert "Product Name: D51B-2U (dual 10G LoM)" in lines
+        if self.conf["type"] == "quanta_t41":
+            assert "Manufacturer: QuantaPlex Computer Inc" in lines
+            assert "Product Name: QuantaPlex T41S-2U" in lines
+        if self.conf["type"] == "dell_c6320":
+            assert "Manufacturer: Dell Inc" in lines
+            assert "Product Name: PowerEdge C6320" in lines
+        if self.conf["type"] == "dell_r630":
+            assert "Manufacturer: Dell Inc" in lines
+            assert "Product Name: PowerEdge R630" in lines
+        if self.conf["type"] == "s2600kp":
+            assert "Manufacturer: EMC" in lines
+            assert "Product Name: S2600KP" in lines
+        if self.conf["type"] == "s2600tp":
+            assert "Manufacturer: EMC" in lines
+            assert "Product Name: S2600TP" in lines
+        if self.conf["type"] == "s2600wtt":
+            assert "Manufacturer: EMC" in lines
+            assert "Product Name: S2600WTT" in lines
```

Comment:
@JuneZhou19 , test.yml always host the type as "quanta_d51" by default, so you can keep one node type instead of 7.
Take coverage on several node type into consideration, I suggest adding test with node type changed, by doing so, you can easily add another test once we support other nodes.